### DISCUSSION TITLE
Getting  "connect ETIMEDOUT 104.16.21.35:443" on corporate network.

### DISCUSSION THREAD

**Discussion by GanizaniSitara** [lowdefy/lowdefy#929] [Questions and Help]
Hi, trying to run up lowdefy on corporate network (js from lowdefy.com explicitly blocked by security). I'm getting connect ETIMEDOUT 104.16.21.35:443, the IP seems to be a cloudflare server. Is it possible to host lowdefy without internet connectivity? Any ideas of what to look for? Thanks.

**Comment by SamTolmay** [comment]
Hi @GanizaniSitara 

Lowdefy pulls in the blocks it uses from a CDN, hosted by Cloudflare.

You can host your own blocks, and configure the Lowdefy app to use those blocks instead.

You will need to build the Lowdefy blocks, and then host the build artifacts on a static file server.

You can follow this discussion with details how to do it: https://github.com/lowdefy/lowdefy/discussions/668.
The "Blocks Server" section at https://docs.lowdefy.com/cli also has some more details.

If you have any other questions, please ask them on discussion #668 so we can keep all the information on this in one place 😀
